Requirement already satisfied: redis in /home/calitos4488/.local/lib/python3.7/site-packages (3.5.3)
